src/Npgsql/NpgsqlConnector.cs

```diff
@@ -1830,7 +1836,7 @@ void PerformKeepAlive(object? state)
                 Log.Error("Keepalive failure", e, Id);
                 try
                 {
-                    Break(e);
+                    Break(new NpgsqlException("Connection is broken due to keepalive failure, check logs for more information."));
```

Comment:
```suggestion
                    Break(new NpgsqlException("The connection was broken due to a keepalive failure, check the logs for more information."));
```

Reply:
As we're returning the original exception, I don't see a reason in adding the message about checking the logs.